peripherals/rcc/rcc_u5.yaml

```
@@ -0,0 +1,721 @@
+    MSIBIAS:
+      Continuous: [0, "MSI bias continuous mode (clock accuracy fast settling time)"]
+      Sampling: [1, "MSI bias sampling mode when the regulator is in range 4, or when the device is in Stop 1 or Stop 2 (ultra-low-power mode)"]
+    MSICAL?: [0, 0x1F]
+  ICSCR2:
+    MSITRIM?: [0, 0x1F]
+  ICSCR3:
+    HSITRIM: [0, 0x1F]
+    HSICAL: [0, 0xFFF]
+  CRRCR:
+    HSI48CAL: [0, 0x1FF]
+  CFGR1:
+    MCOPRE:
+      Div1: [0, "MCO divided by 1"]
+      Div2: [1, "MCO divided by 2"]
+      Div4: [2, "MCO divided by 4"]
+      Div8: [3, "MCO divided by 8"]
+      Div16: [4, "MCO divided by 16"]
+    MCOSEL:
+      None: [0, "MCO output disabled, no clock on MCO"]
+      SYSCLK: [1, "SYSCLK system clock selected"]
+      MSIS: [2, "MSIS clock selected"]
+      HSI16: [3, "HSI16 clock selected"]
+      HSE: [4, "HSE clock selected"]
+      PLL: [5, "Main PLL clock pll1_r_ck selected"]
+      LSI: [6, "LSI clock selected"]
+      LSE: [7, "LSE clock selected"]
+      HSI48: [8, "Internal HSI48 clock selected"]
+      MSIK: [9, "MSIK clock selected"]
+    STOPKERWUCK:
+      MSIK: [0, "MSIK oscillator automatically enabled when exiting Stop mode or when a CSS on HSE event occurs."]
+      HSI16: [1, "HSI16 oscillator automatically enabled when exiting Stop mode or when a CSS on HSE event occurs."]
+    STOPWUCK:
+      MSIS: [0, "MSIS oscillator selected as wake-up from stop clock and CSS backup clock"]
+      HSI16: [1, "HSI16 oscillator selected as wake-up from stop clock and CSS backup clock"]
+    SWS:
+      _read:
+        MSIS: [0, "MSIS oscillator used as system clock"]
+        HSI16: [1, "HSI16 oscillator used as system clock"]
+        HSE: [2, "HSE used as system clock"]
+        PLL: [3, "PLL pll1_r_ck used as system clock"]
+    SW:
+      MSIS: [0, "MSIS selected as system clock"]
+      HSI16: [1, "HSI16 selected as system clock"]
+      HSE: [2, "HSE selected as system clock"]
+      PLL: [3, "PLL pll1_r_ck selected as system clock"]
+  CFGR2:
+    APB2DIS:
+      Enabled: [0, "APB2 clock enabled, distributed to peripherals according to their dedicated clock enable control bits"]
+      Disabled: [1, "APB2 clock disabled"]
+    APB1DIS:
+      Enabled: [0, "APB1 clock enabled, distributed to peripherals according to their dedicated clock enable control bits"]
+      Disabled: [1, "APB1 clock disabled"]
+    AHB2DIS2:
+      Enabled: [0, "AHB2_2 clock enabled, distributed to peripherals according to their dedicated clock enable control bits"]
+      Disabled: [1, "AHB2_2 clock disabled"]
+    AHB2DIS1:
+      Enabled: [0, "AHB2_1 clock enabled, distributed to peripherals according to their dedicated clock enable control bits"]
+      Disabled: [1, "AHB2_1 clock disabled"]
+    AHB1DIS:
+      Enabled: [0, "AHB1 clock enabled, distributed to peripherals according to their dedicated clock enable control bits"]
+      Disabled: [1, "AHB1 clock disabled"]
+    DPRE:
+      Div1: [0, "DCLK not divided"] # same for [0, 3]
```

```suggestion
      Div1: [-1, "DCLK not divided"] # same for [0, 3]
```
This makes `Div1` default value for reading.